.devcontainer/devcontainer.json

```diff
@@ -60,5 +60,13 @@
     "image": "mcr.microsoft.com/vscode/devcontainers/python:3.12",
     "name": "PyTado",
     "postCreateCommand": "python3 -m venv venv && . venv/bin/activate && pip install --upgrade pip && pip install -e '.[all]' && pre-commit install",
-    "updateContentCommand": "python3 -m venv venv && . venv/bin/activate && pip install --upgrade pip && pip install -e '.[all]' && pre-commit install"
+    "updateContentCommand": "python3 -m venv venv && . venv/bin/activate && pip install --upgrade pip && pip install -e '.[all]' && pre-commit install",
+    "runArgs": [
+        "--userns=keep-id"
```

Comment:
This isnt working for me

```
[2024-12-30T08:02:16.397Z] docker: --userns: invalid USER mode.
See 'docker run --help'.
```

Reply:
I will change it, sadly i have to use podman rootful!